tests/testthat/test_array_manipulation.r

```diff
@@ -5,6 +5,17 @@ library(data.table)
 library(dplyr)
 options('stringsAsFactors' = FALSE)
 
+# copy of the function repr_matrix_df.r
+.char_fallback <- function(char, default) {
```

Comment:
can you really not use the one from the package? maybe do ellip.h <- IRkernel:::ellip.h` (and so on)?